Makefile

```diff
@@ -87,8 +89,25 @@ e2e-install-prerequisites:
 	# Download and install bats
 	curl -sSLO https://github.com/bats-core/bats-core/archive/v${BATS_VERSION}.tar.gz && tar -zxvf v${BATS_VERSION}.tar.gz && sudo bash bats-core-${BATS_VERSION}/install.sh /usr/local
 
-e2e-setup-kind:
-	./scripts/ci-e2e-kind.sh
+.PHONY: install-soak-prerequisites
+install-soak-prerequisites:
+	# Download and install kubectl
+	curl -LO https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_VERSION}/bin/linux/amd64/kubectl && chmod +x ./kubectl && sudo mv kubectl /usr/local/bin/
+	# Download and install bats
+	curl -sSLO https://github.com/bats-core/bats-core/archive/v${BATS_VERSION}.tar.gz && tar -zxvf v${BATS_VERSION}.tar.gz && sudo bash bats-core-${BATS_VERSION}/install.sh /usr/local
+	# Download and install node-shell
+	curl -LO https://github.com/kvaps/kubectl-node-shell/raw/master/kubectl-node_shell && chmod +x ./kubectl-node_shell && sudo mv ./kubectl-node_shell /usr/local/bin/kubectl-node_shell
```

Comment:
Instead of a new target without `kind` could we just reuse `e2e-install-prerequisites` and install node-shell?

```bash
.PHONY: install-soak-prerequisites
install-soak-prerequisites: e2e-install-prerequisites
	# Download and install node-shell
	curl -LO https://github.com/kvaps/kubectl-node-shell/raw/master/kubectl-node_shell && chmod +x ./kubectl-node_shell && sudo mv ./kubectl-node_shell /usr/local/bin/kubectl-node_shell
```

Since this is used only for CI, having kind installed should be fine. WDYT?

Reply:
I was going back and forth on this. On one hand we are unnecessarily installing kind when not needed but on the other hand we are also creating separate target which does almost same the job. I am fine with above approach as well. I'll update target to this.